services/kuber/templates/autoscaler-adapter.goyaml

```diff
@@ -19,11 +23,13 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      app: autoscaler-adapter-{{ .ClusterID }}
+      app.kubernetes.io/part-of: claudie
+      app.kubernetes.io/name: autoscaler-adapter-{{ .ClusterID }}
   template:
     metadata:
       labels:
-        app: autoscaler-adapter-{{ .ClusterID }}
+        app.kubernetes.io/part-of: claudie
+        app.kubernetes.io/name: autoscaler-adapter-{{ .ClusterID }}
```

Comment:
I think this shouldn't be templated, it should just be `autoscaler-adapter`
But actually, instead of that,  I'd go with `app.kubernetes.io/name: cluster-autoscaler` for everything